apps/client/src/features/rundown/event-block/composite/EventBlockChip.tsx

```diff
@@ -75,3 +78,51 @@ function EventUntil(props: EventUntilProps) {
     </Tooltip>
   );
 }
+
+interface EventReportProps {
+  className: string;
+  id: string;
+  duration: number;
+}
+
+function EventReport(props: EventReportProps) {
+  const { className, id, duration } = props;
+  const { data } = useReport();
+  const currentReport: OntimeEventReport | undefined = data[id];
+
+  const [value, isOver] = useMemo(() => {
+    if (!currentReport) {
+      return [null, false];
+    }
+
+    const { startedAt, endedAt } = currentReport;
+    if (!startedAt || !endedAt) {
+      return [null, false];
+    }
+
+    const actualDuration = endedAt - startedAt;
+    const difference = actualDuration - duration;
+    const absDifference = Math.abs(difference);
+
+    if (absDifference < MILLIS_PER_SECOND) {
+      return ['==', false]; //TODO: how to symbolize ontime?
```

Comment:
good question, how about just a checkmark? either unicode ✓ or a svg?

Reply:
I will try that